doc/source/whatsnew/v0.24.0.txt

```diff
@@ -225,7 +225,7 @@ Plotting
 Groupby/Resample/Rolling
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
--
+- Bug in :func:`pandas.core.groupby.first` and :func:`pandas.core.groupby.last` with ``as_index=False`` leading to the loss of timezone information (:issue:`15884`)
```

Comment:
I think the `:func:` links should be `pandas.core.groupby.GroupBy.first` (and likewise for `last`)

Reply:
Yep, you're right, thanks!